src/apps/workspaces/domain/workspace.py

```diff
@@ -126,7 +127,27 @@ class WorkspaceManager(InternalModel):
     last_seen: datetime.datetime
     is_pinned: bool = False
     applets: list[WorkspaceManagerApplet] | None = None
+    titles: list[str] | None
     title: str | None
+    status: InvitationStatus
+
+    @validator("titles", pre=True)
+    def get_titles(cls, value):
+        if len(value) > 0:
+            value = [
+                StringEncryptedType(Unicode, get_key).process_result_value(v, dialect=PGDialect_asyncpg.name)
+                for v in value
+                if v is not None
+            ]
```

Comment:
You can do type cast in the query and sqlalchemy decrypts the values itself:
```
.cast(ARRAY(StringEncryptedType(Unicode, get_key)))
```

Reply:
great! thanks

Reply:
done!